package.json

```diff
@@ -198,7 +199,8 @@
     "lint-fix": "eslint --ext .js,.ts,.tsx --max-warnings 0 --fix .",
     "postinstall": "patch-package --error-on-fail && jetify",
     "validate": "npm ls",
-    "start": "make dev"
+    "start": "make dev",
+    "prepare": "sh ./prepare_sdk.sh"
```

Comment:
Let's put this in the SDK project's package.json file.

Reply:
Done